manifests/2/2BrightSparks/SyncBackProX64/10.2.68.0/2BrightSparks.SyncBackProX64.locale.en-US.yaml

```
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: 2BrightSparks.SyncBackProX64

```

```suggestion
PackageIdentifier: 2BrightSparks.SyncBackPro
```
You'll have to change the `SyncBackProX64` folder to `SyncBackPro` ...
... and update the file names as well.

You can do this by pressing the `.` (dot) key on your keyboard which will open up Visual Studio Online.